ws-client/src/transport.rs

```diff
@@ -233,7 +228,6 @@ impl<'a> WsTransportClientBuilder<'a> {
 	/// Sets the timeout to use when establishing the TCP connection.
 	///
 	/// The default timeout is 10 seconds.
-	// TODO: design decision: should the timeout not be handled by the user?
```

Comment:
Removed this comment because it's exposed to the user but these methods are not used. It's exposed via the `WsConfig`.